apps/ensnode/src/handlers/Registry.ts

```diff
@@ -128,8 +129,14 @@ export const makeRegistryHandlers = (ownedName: OwnedName) => {
 
           // TODO: implement sync rainbow table lookups
           // https://github.com/ensdomains/ens-subgraph/blob/master/src/ensRegistry.ts#L111
-          const labelName = encodeLabelhash(label);
-          const name = parent?.name ? `${labelName}.${parent.name}` : labelName;
+          let labelName = encodeLabelhash(label);
```

Comment:
```suggestion
          let unknownLabel = encodeLabelhash(label);
```

When we encode a labelhash, it generates an "unknown label".

Reply:
this code should minimc the subgraph and the diff should look like:

```ts
const healedLabel = await labelHealing.heal(label);
const labelName = healedLabel || encodeLabelhash(label);
const name = parent?.name ? `${labelName}.${parent.name}` : labelName;

await context.db.update(schema.domain, { id: domain.id }).set({
  name,
  // only set domain.labelName iff label is healed and indexable
  // https://github.com/ensdomains/ens-subgraph/blob/master/src/ensRegistry.ts#L113
  labelName: healedLabel,
});
```